### ISSUE TITLE
Unclear flow of commands available to a user.

### ISSUE THREAD

**Issue by eujingsen** [eujingsen/pe#2]
While this may not be an issue for the creator or any experienced user of AlphaNUS, the commands available for a user at any point in time is not clearly defined and there is no flow to the commands. Basically, any command can be used at any point in time and there are little to no prompts as to what you can do after a command is inputted outside of typing the help command and showing a long list of what you can do.
Little help is given with regards to the current course of action that a user can do after any command.

To summarise, it is not user friendly to someone who has little to no experience using alphaNUS.
